File: src/br/com/globalcode/android/ClientFragment.java
package br.com.globalcode.android;

import android.app.Fragment;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class ClientFragment extends Fragment {

	@Override
	public View onCreateView(LayoutInflater inflater, ViewGroup container,
			Bundle savedInstanceState) {
		
		View v = inflater.inflate(R.layout.client_fragment_layout, container, false);
		return v;
	}
}

File: src/br/com/globalcode/android/ClientFragment.scala
package br.com.globalcode.android

import android.app.Fragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class ClientFragment extends Fragment {

	override def onCreateView(inflater: LayoutInflater, container: ViewGroup,
			savedInstanceState: Bundle) : View = {
		inflater.inflate(R.layout.client_fragment_layout, container, false);
	}
}